README.md

```diff
@@ -164,6 +167,51 @@ yotiClient.getActivityDetails(token).then((activityDetails) => {
 
 [Example reference](examples/profile/index.js)
 
+
+## Programmatic QR code creation
+
+Dynamic attribute list generation is a way of requesting non-static attribute lists for your application.
```

Comment:
Does this still make sense as below the term dynamic policy is now used?

Reply:
I'll change this to:
> Dynamic Policy generation is a way of requesting non-static attribute lists for your application.